app/helpers/users_helper.rb

```diff
@@ -0,0 +1,12 @@
+module UsersHelper
+  USER_COMMENTS_PARTIAL = "users/comments_section".freeze
+  COMMENTS_LOCKED_PARTIAL = "users/comments_locked_cta".freeze
+
+  def user_comments_section(_comments)
```

Comment:
It seems like we don't need passing `_comments` here. `@comments` variable will be accessible in the partials anyway.